usaspending_api/common/views.py

```diff
@@ -94,3 +95,16 @@ def get_context_data(self, **kwargs):
         # Add in the markdown to the context, for use in the template tags
         context["markdown"] = self.markdown
         return context
+
+
+# placeholder for retired URLs
+@method_decorator(removed, name="retrieve")
+class RemovedEndpointView(viewsets.ViewSet):
+    """
+        Placeholder for endpoints that have been removed
+    """
```

Comment:
When the docstrings are short, one-line is preferable:

{code:python}
"""Placeholder for endpoints that have been removed"""
{code}